src/components/Question/QuestionList.jsx

```diff
@@ -0,0 +1,11 @@
+import Question from './Question'
+
+const QuestionList = ({ questions, currentQuestionIndex, handleAnswerClick }) => (
```

Comment:
Similar to the case of the "Answer" component.  Typically, the term "list" added to a component's name is used when there is iteration of elements inside it.